lua/entities/gmod_tardis/modules/cl_music.lua

```diff
@@ -22,10 +22,12 @@ TARDIS:AddSetting({
 })
 
 function ENT:StopMusic()
-	if IsValid(self.music) then
+    if IsValid(self.music) then
+        if self.music:GetState() == GMOD_CHANNEL_PLAYING then
+			TARDIS:Message(LocalPlayer(), "Music stopped.")
+		end
 		self.music:Stop()
-		self.music=nil
-	end
+    end
```

Comment:
There is a mix of tabs and spaces in the indenting used here. This should be all using tabs. You can see this in VS Code by pressing Ctrl+Alt+P and selecting `View: Toggle Render Whitespace`

![image](https://user-images.githubusercontent.com/2363642/148763487-8b44d481-f54b-48fd-9df5-486893f277fb.png)